src/plugins/links/public/components/dashboard_link/dashboard_link_component.tsx

```diff
@@ -97,9 +104,9 @@ export const DashboardLinkComponent = ({
   /**
    * Dashboard-to-dashboard navigation
    */
-  const { loading: loadingOnClickProps, value: onClickProps } = useAsync(async () => {
+  const onClickProps = useMemo(() => {
```

Comment:
This was accidentally left `async` in https://github.com/elastic/kibana/pull/167928 when it should have been removed.